training/src/main/java/wolox/training/models/Client.java

```diff
@@ -0,0 +1,71 @@
+package wolox.training.models;
+
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+public class Client {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String username;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @OneToMany(mappedBy = "client")
```

Comment:
Like I said in the Book class, this is a ManyToMany relationship. I would put a mappedBy solution here and set the owning side of the relationship (the one that says if its ManyToMany or whatever) on th side of the Book class.